### PR TITLE
Adding test in the root package in order to test the run command

### PR DESCRIPTION
**Pull request by fokion**
Adding test for the `run` command , running local tests that we currently only run in E2E.

### CODE REVIEW

**Comment by fokion** [comment]
https://github.com/ovh/venom/pull/708

File: venom.go [hunk 1]
```diff
@@ -35,6 +36,15 @@ func OSExit(exitCode int) {
 	}
 }
 
+// OSExit is a wrapper for os.Exit
+func (v *Venom) OSExit(exitCode int) {
+	if v.InTestMode {
```
[the remark this answers is not in the document]
Comment (fokion) [reply venom.go:41]: do you know how can you use this in the unit test ?

[the remark this answers is not in the document]
Comment (fokion) [reply venom.go]: will do

File: venom.go [hunk 2]
```diff
@@ -73,6 +83,7 @@ type Venom struct {
 	StopOnFailure bool
 	HtmlReport    bool
 	Verbose       int
+	InTestMode    bool
```
[the remark this answers is not in the document]
Comment (fokion) [reply venom.go]: will do

**Review by yesnault — APPROVED** [review]

**Pull request merged**